app/assets/javascripts/draft_form_selectors.coffee

```diff
@@ -322,6 +322,34 @@ $(document).ready ->
       when 'NotAvailable'
         $parent.find('.doi-fields.not-available').show()
         $parent.find('.doi-fields.not-available select').val('Not Applicable')
+        
+  # Handle License_URL Available selector
+  $('.license-available-select').change ->
+    $parent = $(this).parents('.license-group')
+    $parent.find('.license-text-fields').hide()
+    $parent.find('.license-url-fields').hide()
+
+    # Clear license-available-select radio buttons
+    # that aren't the one just selected
+    $parent.find('input').not("##{$(this).attr('id')}").prop 'checked', false
+
+    # show the selected fields
+    switch $(this).val()
+      when 'LicenseText'
+        $parent.find('.license-text-fields').show()
+        $parent.find('.license-text-fields').find('label').addClass('eui-required-o')
+        $parent.find('.license-url-fields').find('input, select').val ''
+      when 'LicenseUrl'
+        $parent.find('.license-url-fields').show()
+        $parent.find('.license-text-fields').find('textarea').val ''
+
+  # Handle License Text required icon
+  $('#draft_use_constraints_license_text').change ->
```

Comment:
I was asking about his because we have a separate file that handles the required icons, and we _should_ try and have it all work through that functionality. I think this can be removed, with a suggestion to the license text field below

Reply:
Removed this javascript.   The problem was that the required icon handling required a blur event.  Added the blur triggers to the radio buttons.